gammapy/modeling/serialize.py

```diff
@@ -7,7 +7,12 @@
 # TODO: move this elsewhere ?
 DATASETS = Registry([MapDataset, SpectrumDataset, FluxPointsDataset])
 
-__all__ = ["models_to_dict", "dict_to_models", "dict_to_datasets", "datasets_to_dict"]
+__all__ = [
```

Comment:
I think there's a difference in line length in the black formatter?
Can you please try this?
```
pip install -U black
black gammapy/modeling/serialize.py gammapy/catalog/fermi.py
```
It should revert this unrelated linebreak  change which is not according to our coding format.